instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlStatementSanitizerTest.java

```diff
@@ -58,6 +58,36 @@ void veryLongSelectStatementsAreOk() {
     assertThat(result).isEqualTo(expected);
   }
 
+  @Test
+  void changeSchemaStatementsAreOk() {
+    String createTableStatement = "CREATE TABLE `table`";
```

Comment:
Some databases also support `CREATE TABLE IF NOT EXISTS table` which will have `IF` as table name, wdyt of handling that also or at least making it so that it would have table name as `null`?

Reply:
Added a test case for it. It returns the correct table name.

Reply:
Fixed.